### DISCUSSION TITLE
phcode opens unwanted folders

### DISCUSSION THREAD

**Discussion by lapindejade** [phcode-dev/phoenix#1896] [Q&A]
Hello, since I installed phcode under Manjaro Linux, when I click on my folders it opens them by default. I can't change that and it's a important problem, because I can't reach my documents and do my work. How can I remedy this please?

**Comment by lapindejade** [comment]
I finally succeeded to uninstall it.

**Reply by abose** [reply]
Thanks for reporting the issue. 

Fixed by https://github.com/phcode-dev/phoenix-desktop/pull/556 . will ve available in a few days.
